src/target/stm32mp15.c

```diff
@@ -118,6 +126,51 @@ static bool stm32mp15_ident(target_s *const target, const bool cortexm)
 	return true;
 }
 
+static bool stm32mp15_cmd_swo(target_s *target, int argc, const char **argv)
+{
+	(void)argc;
+	(void)argv;
+	/* TODO: argv parsing for mode and baudrate */
+	adiv5_access_port_s *const ap_apbd = (adiv5_access_port_s *)target->target_storage;
```

Comment:
As with the other places this `apbd` naming has come up, we do not know where this `d` is coming from in the naming - please either name by the AP number this AP represents, or just say `ap_apb` - if it's supposed to mean "debug", that's already implied by it being the bus attached to the AP, but those busses aren't some special thing.. they are  normal APB's and AHB's, so including the `d` in the nomenclature just muddies things in our opinion. Readers not familiar with things would try searching for an `APBD` and get no results which is counter-productive, for example.